content/docs/run-api-server/ingestion.mdx

```diff
@@ -0,0 +1,292 @@
+
+
+## Ingestion Types
+
+There are two primary ingestion use-cases for Horizon operations:
+
+  - ingesting **live** data to stay up to date with the latest, real-time changes to the Stellar network, and
+  - ingesting **historical** data to peek how the Stellar ledger has changed over time
+
+### Ingesting Live Data
+
+Though this option is disabled by default, in this guide we've [assumed](./configuring.mdx) you turned it on. If you haven't, pass the `--ingest` flag or set `INGEST=true` in your environment.
+
+For a serious setup, **we highly recommend having more than one live ingesting instance**, as this makes it easier to avoid downtime during upgrades and adds resilience to your infrastructure, ensuring you always have the latest network data.
+
+### Ingesting Historical Data
+
+Providing API access to historical data is facilitated by a Horizon subcommand:
+
+<CodeExample>
+
+```
+stellar-horizon db reingest range <start> <end>
+```
+
+</CodeExample>
+
+_(The command name is a bit of a misnomer: you can use `reingest` both to ingest new ledger data and reingest old data.)_
+
+You can run this process in the background while your Horizon server is up. It will continuously decrement the `history.elder_ledger` in your `/metrics` endpoint until the `<end>` ledger is reached and the backfill is complete. If Horizon receives a request for a ledger it hasn't ingested, it returns a 503 error and clarify that it's `Still Ingesting` (see [below](#some-endpoints-are-not-available-during-state-ingestion)).
+
+#### Deciding on how much history to ingest
+(TODO: Think of a better section name; pls suggest something.)
+
+You should think carefully about the amount of ingested data you'd like to keep around. Though the storage requirements for the entire Stellar network are substantial, **most organizations and operators only need a small fraction of the history** to fit their use case. For example,
+
+  * If you just started developing a new application or service, you can probably get away with just doing live ingestion, since nothing you do requires historical data.
+
+  * If you're moving an existing service away from reliance on SDF's Horizon, you likely only need history from the point at which you started using the Stellar network.
+
+  * If you provide temporal guarantees to your users--a 6-month guarantee of transaction history like some online banks do, or history only for the last thousand ledgers (see [below](#managing-storage)), for example--then you similarly don't have heavy ingestion requirements.
+
+Even a massively-popular, well-established service such as Coinbase probably doesn't need full history to service its users. It will, however, likely need it to be a [Full Validator](../run-core-node/index.mdx#full-validator) with published history archives.
```

Comment:
```suggestion
Even a massively-popular, well-established service probably doesn't need full history to service its users. It will, however, likely need it to be a [Full Validator](../run-core-node/index.mdx#full-validator) with published history archives.
```

Reply:
I don't think we should call out Coinbase, especially since we don't know what they actually need.